package.json

```diff
@@ -7,6 +7,7 @@
   "main": "src/gitpodify.js",
   "scripts": {
     "build": "yarn clean && npx tsc && yarn webpack",
+    "build:safari": " yarn install && yarn build && yarn package && xcrun safari-web-extension-converter . --app-name Gitpod --bundle-identifier io.gitpod.Gitpod",
```

Comment:
I'd keep `yarn install` as a separate step and rather not include it in this build script.

Reply:
Cool, I'll update it. And correct Xcode 12 is required for the `safari-web-extension-converter`. I'll update the readme to note this.

Reply:
Fixed 😄